### ISSUE TITLE
Equivalent of --skip-existing from s3cmd

### ISSUE THREAD

**Issue by jimmywan** [bloomreach/s4cmd#135]
Does this exist?

**Comment by navinpai** [comment]
No, this does not exist in s4cmd today... Making this issue a feature request, but it might be a while since there's a pending backlog already. Also, if you are comfortable, feel free to add this feature and send in a pull request! :)
